workspace_functions.go

```diff
@@ -0,0 +1,23 @@
+package databricks
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/databricks/databricks-sdk-go/httpclient"
+)
+
+// CurrentWorkspaceId returns the workspace ID of the workspace that this client is
+// connected to.
+func (w *WorkspaceClient) CurrentWorkspaceId(ctx context.Context) (int64, error) {
```

Comment:
Nit: for generated code we use `Id` but for manually written code we should stick to `ID` (see https://google.github.io/styleguide/go/decisions.html#initialisms).

Reply:
Thanks for the tip!